src/plugins/three/DebugTilesPlugin.js

```diff
@@ -108,31 +141,48 @@ export class DebugTilesPlugin {
 		// register events
 		this._onLoadTileSetCB = () => {
 
-			this._initExtremes();
+			if ( this.enabled ) {
+
+				this._initExtremes();
+
+			}
 
 		};
 
 		this._onLoadModelCB = ( { scene, tile } ) => {
 
-			this._onLoadModel( scene, tile );
+			if ( this.enabled ) {
+
+				this._onLoadModel( scene, tile );
+
+			}
 
 		};
 
 		this._onDisposeModelCB = ( { tile } ) => {
 
+			// Note that we want to cleanup memory even if the plugin is disabled.
```

Comment:
This comment seems out of date - since we remove the events when the plugin is disabled this function will no longer be called.